app/views/green_lanes/check_your_answers/show.html.erb

```diff
@@ -11,14 +11,16 @@
                edit_path: 'EDIT GOOD DETAILS PAGE'
     %>
 
-    <h2 class="govuk-heading-m">Category exemptions</h2>
+    <% unless @category_one_assessments.empty? || @answers.nil? %>
```

Comment:
two things
- We've been named it using numbers until now, to be consistent let's call it:
`@category_1_assessments`

-  it looks the logic is same as this one (that should be more readable):

`if  @category_one_assessments.present? && @answers.present?`

Reply:
I can change this but it was used anyways @category_one_assessments.

Im not sure that logic would work as there could be cat 1 assessments but the user hasnt been shown them because at least one of them has no exemptions hence adding the or clause